### DISCUSSION TITLE
Drop OpenStack provisioning

### DISCUSSION THREAD

**Discussion by StykMartin** [beaker-project/beaker#99] [Ideas]
I would propose to drop OpenStack provisioning[0] from Beaker.
I'm not aware of any user and it is huge pain to maintain it. We had to break casual workflow to fit it there.

If there will be no objection I will be working on patching it out before I will start working on Python 3 transition in the server.

[0] https://beaker-project.org/dev/proposals/dynamic-virtualization.html

**Comment by JohnVillalovos** [comment]
We do not use this feature and are not opposed to it being removed.

**Comment by renanrodrigo** [comment]
Isee that removing it now will ease up the development.
Are there any plans to re-integrate it (or integrate to any other cloud) in the future, in a better shape?

**Reply by StykMartin** [reply]
It may happen that we will reintroduce this but in a completely different fashion. Maybe GCP boxes attached to remote Beaker or something like that.

**Reply by renanrodrigo** [reply]
Sounds reasonable.
I can see how it would help in some cloud-testing scenarios, and agree that this is future and definitely refashionable.

Right now, no opposition to drop the support (:
